themes/src/main/resources/theme/base/login/resources/js/authChecker.js

```diff
@@ -1,5 +1,7 @@
 const CHECK_INTERVAL_MILLISECS = 2000;
+const AUTH_SESSION_TIMEOUT_MILLISECS = 1000;
```

Comment:
Is 1 second sufficient? I wonder if there are some corner cases when 1 second is not sufficient (slow web page, lots of tabs refreshed at the same time). Will it makes sense for example to repeat this 10 times with 1 second interval, which would mean 10 seconds in total? Can be probably achieved with something like `setInterval` or with similar mechanism like used for checking `KEYCLOAK_SESSION` cookie (the function itself calls another `setTimeout`).

Reply:
Another difficult decision, checking for X times every 1 second can cover the use case where a tab takes up to X seconds to load the page and set the cookie. The problem with this is that it would require modifying `WaitUtils.pause(2000);` in the `MultipleTabsLoginTest` tests to at least 10 seconds, which might be a bit too much.

Reply:
For tracking: We had private discussion with @graziang and based on that, I am closing this point for now. Hopefully 1 second is sufficient for most of the cases (or all). We can revisit in a follow-up if issue can be still reproduced for some very corner cases though...